### DISCUSSION TITLE
Typeahead example?

### DISCUSSION THREAD

**Discussion by subproject22** [radix-ui/primitives#3913] [Help]
Hey everyone. 

Anyone willing to share how to use declared `Select` Typeahead functionality? Do I need to bind it's `value` to my custom input or something? What to do with trigger? Hide it?

**Comment by manfromanotherland** [comment]
Setting `textValue` on `<Select.Item>` with complex children enabled the typeahead functionality for me. In my case, I was running into the issue because I have icons in my Select items, so typeahead would not work.

Alternatively, you can also wrapped the options’ text with `<Select.ItemText>` and that will be used for typeahead too.
